### ISSUE TITLE
Enable floating offers for Cash at ATM payment method

### ISSUE THREAD

**Issue by woodser** [haveno-dex/haveno#648]
This issue requests enabling floating offers for the Cash at ATM method.

Currently only fixed price offers are supported, to prevent cash amounts from exceeding e.g. $500.

The current trigger price could be used, but then the user would need to manually calculate the market price to not exceed.

**Comment by woodser** [comment]
For floating offers using the Cash at ATM payment method, I think the offer amount and min amount don't need to be adjusted to the nearest $10 increment like with fixed price offers.

**Comment by superhans65** [comment]
> Currently only fixed price offers are supported, to prevent cash amounts from exceeding e.g. $500.

I see this (ie exceeding $500) as less of an issue than the lack of floating offers and trigger protection. But maybe I am missing something? Couldn't the buyer/taker just specify $500 or whatever lower amount he/she knows will work?

**Comment by XMRfamily** [comment]
You should include this option for Cash at branch (in usa some banks allow cash deposits for users at branches but not at atms)

**Comment by charcharchar** [comment]
> > Currently only fixed price offers are supported, to prevent cash amounts from exceeding e.g. $500.
> 
> I see this (ie exceeding $500) as less of an issue than the lack of floating offers and trigger protection. But maybe I am missing something? Couldn't the buyer/taker just specify $500 or whatever lower amount he/she knows will work?

I agree with this, this payment method is common in Australia and would be good to have floating price especially early on with low amount of users as offers will be online for longer and high chance of price changing. What if a lower limit than 500 is hard set as a quick fix, so there is plenty of room for the price to move without going over the 500 limit? Another option would be to use the 'Deactivate offer if market price is above'[set optional trigger price] feature but hard code in a price based on region? I would assume different countries have different price limits. This payment method is often called or referred to as cardless cash in Australia.

**Comment by charcharchar** [comment]
> You should include this option for Cash at branch (in usa some banks allow cash deposits for users at branches but not at atms)

Isn't that covered by the 'Cash Deposit' payment method?

**Comment by charcharchar** [comment]
The lack of floating price not enabled here is killing the use of this really good payment method for Australians for cash at atm. Please consider enabling it. Thanks

**Comment by charcharchar** [comment]
> > Currently only fixed price offers are supported, to prevent cash amounts from exceeding e.g. $500.
> 
> I see this (ie exceeding $500) as less of an issue than the lack of floating offers and trigger protection. But maybe I am missing something? Couldn't the buyer/taker just specify $500 or whatever lower amount he/she knows will work?

Agreed seems like a bit of a non-issue. If it happens that above max is taken, the traders could probably agree between themself how to handle or could cancel and redo again. the way it is now with low users people cant trust putting up a fixed offer because it could be a bad price for the seller or crap price for the buyer so is never taken and needs to be cancelled.
